sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2020-11-10)
 
 This is the first preview of the next generation of `Microsoft.Azure.WebJobs.Extension.Storage` which has been integrated with latest Azure Storage SDK that follows the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
```

Comment:
Should this say `.Storage.Queues`?

Reply:
the sentence refers to the past where `Microsoft.Azure.WebJobs.Extension.Storage` wasn't split.

Reply:
The changelog mentions the split further down.